Automattic/jetpack
# JS Components: add Protect Icon

JS Components: add Protect Icon

Fixes #

#### Changes proposed in this Pull Request:

* JS Components: add Protect Icon

#### Jetpack product discussion


p1651070747691529-slack-C02TQF5VAJD

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* With Storybook
* Go to JS Packages / Components / Product Icons
* Take a look at the Protect icon
* Confirm it looks good

<img width="1243" alt="image" src="https://user-images.githubusercontent.com/77539/165755977-1508fb54-9ba3-4919-9a57-012e78199880.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202183657377556
